src/pocketmine/Player.php

```diff
@@ -3647,6 +3647,24 @@ protected function onDeathUpdate(int $tickDiff) : bool{
 		return false; //never flag players for despawn
 	}
 
+	protected function onTotem() : void{
```

Comment:
This method name doesn't make sense -- it contains logic for actually using the totem but the name suggests it contains logic for handling the event. Perhaps `useTotem()` or `consumeTotem()` would be better. 

Also, does this method belong here? Maybe it would be better in the `pocketmine\entity\Human` class.

Reply:
> Also, does this method belong here? Maybe it would be better in the pocketmine\entity\Human class.

I thought that methods like this are written in `pocketmine\Player`